docs/developer-guide/local-development-gotchas.md

```diff
@@ -1,3 +1,24 @@
+# Using DJANGO-DEBUG-TOOLBAR in Dev
+
+The toolbar is enabled for these variable settings which is default in dev.
+```console
+DEBUG = True
+DEBUG_TOOLBAR_ENABLED = True
+```
+Alternative variable settings in files - [`.env`] or [`local.py`]
+# Running DEBUG=False in Dev 
+[`local.py`] file
+```console
+DEBUG = False
+STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
+```
+
+https://docs.djangoproject.com/en/3.2/ref/contrib/staticfiles/#cmdoption-runserver-insecure
+Run
+```console
+python manage.py runserver 0.0.0.0:8000 --insecure
+```
+
```

Comment:
I was sort of expecting these to be listed as sub-sections below the introductory content. Also, not sure that this explains things that well:

> The toolbar is enabled for these variable settings which is default in dev.

Maybe something like:

```
## How do I disable Django's 'debug' mode?

Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:

- Add `DEBUG = False` to `conf/settings/local.py` 
- Add `DEBUG=False` to your `.env` file

## How do I disable Django Debug Toolbar?

If you don't need Django's debugging tools, setting `DEBUG=False` as detailed above will disable Django Debug Toolbar also.

If you still need Django's debugging tools, but don't want to use Django Debug Toolbar for any reason, you have two options for disabling it:

- Add `DEBUG_TOOLBAR_ENABLED = False` to `conf/settings/local.py` 
- Add `DEBUG_TOOLBAR_ENABLED=False` to your `.env` file
```

Reply:
Done